### DISCUSSION TITLE
How can I use quill in my vue3+ts project

### DISCUSSION THREAD

**Discussion by johnluoyx** [vueup/vue-quill#83] [Q&A] [answered]
HI! I plan to use quill in my vue3+ts project, but i don't know how to use native quill in my vue project. Though luckily I find this vue3-quill, the warning which **the project is not recommended to use this for anything serious** worries me. 
Can I use it normally or is this project has been abandoned?
**_How to use native quill in my vue3+ts project?_**

**Comment by johnluoyx** [comment, marked as answer]
I have found a way to use native quill in vue3+ts project, you can find module type in [http://microsoft.github.io/TypeSearch/](url). Then you can encapsulate your own editor.

**Comment by elitedev219** [comment]
![image](https://github.com/user-attachments/assets/49580d19-e18d-414b-beee-5e5b0fca0dda)
I can't reach your url
